packs/spell-effects/spell-effect-companion-enlarge.json

```diff
@@ -0,0 +1,106 @@
+{
+    "_id": "r3SWKHCXxahSV54y",
+    "img": "systems/pf2e/icons/spells/enlarge-companion.webp",
+    "name": "Spell Effect: Companion Enlarge",
```

Comment:
Should this be "Spell Effect: Enlarge Companion"?

Reply:
Oops
Sorry about that - should now be correct.